core/services/ocrcommon/transmitter.go

```diff
@@ -59,15 +82,13 @@ func (t *transmitter) FromAddress() common.Address {
 }
 
 func (t *transmitter) FromAddressForTransaction() common.Address {
```

Comment:
I think you can just axe this method and call `t.keystore.GetRoundRobinAddress` directly wherever this is called. If it returns an error, then something is up, and we can't transmit (bubble up to caller).

Reply:
True!